linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/process_info.js

```diff
@@ -17,26 +25,30 @@ var ProcInfo = (function() {
     }
   }
 
-  function render(data) {
-    _(data).each(function(obj) {
-      var id = obj.name.replace(/\//g, "-");
-      var value = pretty(obj.name, obj.value);
-      $("#"+id).text(value);
+  function render($root, data) {
+    var templateData = _.map(stats, function(stat) {
+      if (stat.dataKey) {
+        var obj = _.find(data, ["name", stat.dataKey]);
+        var value = pretty(obj.name, obj.value);
+        return _.merge({}, stat, {value: value});
```

Comment:
that was my original thinking, although looking at this a second time, it's probably fine to alter stat (and maybe even preferable for less object garbage)